Packs/CortexAttackSurfaceManagement/Scripts/RemediationPathRuleEvaluation/RemediationPathRuleEvaluation.yml

```diff
@@ -0,0 +1,39 @@
+args:
+- description: Alert's Severity
+  name: severity
+- description: Alert's Remote IP
+  name: ip
+- description: Is this in a development environment?
+  name: development_environment
+- description: Is this Cloud Managed?
+  isArray: true
+  name: cloud_managed
+- description: Has a Service Owner been Identified?
+  isArray: true
+  name: service_owner_identified
+- description: Includes Cloud and Xpanse tags
+  isArray: true
+  name: tags
+- description: Externally Detected Providers
+  isArray: true
+  name: providers
+- description: List of Remediation Path Rules for the Alert's Attack Surface Rule
+  isArray: true
+  name: remediation_path_rules
+comment: For a given alert and remediation path rules that are defined for that alert's attack surface rule, this takes each remediation path rule and looks at the rule criteria too see if the rule matches for the given alert. If multiple rules match, it will return the most recently created rule. This assumes that the rules passed in are filtered to correlate with the alert's attack surface rule.
```

Comment:
```suggestion
comment: For a given alert and remediation path rules that are defined for that alert's attack surface rule, this script takes each remediation path rule and looks at the rule criteria to see if the rule matches for the given alert. If multiple rules match, it will return the most recently created rule. This assumes that the rules passed in are filtered to correlate with the alert's attack surface rule.
```